modifiers/limit_tokens.js

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+/**
+ * @module modifiers/limit_tokens
+ */
+
+const safeObject = require('../util/safe_object');
+
+/**
+ * @typedef {Object} LimitTokensOptions
+ * @property {string[]} [allowedTokens] List of MD tokens allowed to render.
+ * @property {Object} [transformTokens] List of MD tokens to transform into something else.
```

Comment:
We should properly type this object, so that users know how to interact with it

Reply:
Updated the type defs.